packages/lit-dev-content/samples/tutorials/reactivity/tutorial.json

```diff
@@ -0,0 +1,37 @@
+{
+  "header": "Reactivity",
+  "difficulty": "Beginner",
```

Comment:
Can we make this intermediate? The concepts are given a light treatment here, but it's definitely not beginner content imo.

Reply:
Done.